### ISSUE TITLE
Hint: Press shift to move elements faster with keyboard

### ISSUE THREAD

**Issue by bvjebin** [excalidraw/excalidraw#251]
When we move an element with keyboard, it is slow. Providing some way to speed it will be useful for power users.

**Comment by Fausto95** [comment]
Just to make sure, did you try holding <kbd>Shift</kbd> while moving any element?

**Comment by bvjebin** [comment]
Yes. Just tried it. Shift makes it faster. But it was not obvious. Some cue would help.

**Comment by vjeux** [comment]
One thing that would be cool is to display a toast if you've been moving the shape long enough without shift. Contributions welcome

**Comment by dwelle** [comment]
I'm planning to implement a hint manager where we'll put stuff like this.

**Comment by lipis** [comment]
This is done already!

**Comment by lipis** [comment]
Oups..
